frontend/packages/dev-console/src/components/svg/SvgBoxedText.tsx

```diff
@@ -72,6 +74,7 @@ export default class SvgBoxedText extends React.PureComponent<SvgBoxedTextProps,
             ry={cornerRadius}
           />
         )}
+        {icon}
```

Comment:
There needs to be `paddingX` between the icon and the text.
You will need to calculate both the width of the icon and the text, then align the two, and finally calculate the overall width to center the label.
You may find it easier to embed a foreign node w/ icon and text spaced using CSS rather than having to perform all the calculations in the SVG space.
If the icon were a fixed width, this would be easier.

Reply:
oops! this is not part of this PR. I was trying and pushed it accidently. :P